<?php

namespace ContainerNUbYRYx;


use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_ZhzArUZService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.zhzArUZ' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.zhzArUZ'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'categoryRepository' => ['privates', 'App\\Repository\\CategoryRepository', 'getCategoryRepositoryService', true],
            'passwordEncoder' => ['services', 'security.password_encoder', 'getSecurity_PasswordEncoderService', true],
            'user' => ['privates', '.errored..service_locator.zhzArUZ.App\\Entity\\User', NULL, 'Cannot autowire service ".service_locator.zhzArUZ": it references class "App\\Entity\\User" but no such service exists.'],
        ], [
            'categoryRepository' => 'App\\Repository\\CategoryRepository',
            'passwordEncoder' => '?',
            'user' => 'App\\Entity\\User',
        ]);
    }
}
